### PR TITLE
MAISTRA-2579: Setup an experimental e2e job

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: prow/config/presets.yaml [hunk 1]
```diff
@@ -74,3 +74,32 @@ presets:
...
+        key: AWS_ACCESS_KEY_ID
+  - name: AWS_SECRET_ACCESS_KEY
+    valueFrom:
+      secretKeyRef:
+        name: aws
+        key: AWS_SECRET_ACCESS_KEY
+  - name: OCP_PULL_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: aws
```
Comment (brian-avery) [inline prow/config/presets.yaml:99]: The OCP_PULL_SECRET is named `aws`?

Comment (brian-avery) [reply prow/config/presets.yaml:99]: Same with Quay

Comment (jwendell) [reply prow/config/presets.yaml:99]: the secret is

Comment (jwendell) [reply prow/config/presets.yaml:99]: I mean, it's one single secret named aws with multiple entries in its data field.

Comment (jwendell) [reply prow/config/presets.yaml:99]: Secret looks like this:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: aws
type: Opaque
stringData:
  OCP_PULL_SECRET: ''
  QUAY_PULL_SECRET: ''
  AWS_ACCESS_KEY_ID: ''
  AWS_SECRET_ACCESS_KEY: ''
  AWS_DEFAULT_PROFILE: ''
```

**Review by brian-avery — APPROVED** [review]

**Pull request merged**